assets/src/js/Components/ArchivePicker/ArchivePicker.js

```diff
@@ -400,6 +414,66 @@ export default function ArchivePicker({view = ADMIN_VIEW}) {
     }
   }, [currentBlock]);
 
+  // Check whether the archive picker modal was opened from the Featured Image button.
+  // Only execute this code in the Editor.
+  useEffect(() => {
+    if (view === EDITOR_VIEW) {
+      const mediaModal = wp.media.frame || wp.media.featuredImage.frame();
+
+      if (mediaModal?.options?.state === 'featured-image') {
+        setIsUserSelectingFeaturedImage(true);
+      } else {
+        setIsUserSelectingFeaturedImage(false);
+      }
+    }
+  }, [view]);
+
+  // Hide the modal toolbar when the Greenpeace Media tab is active.
+  // The toolbar is not used on this tab, since the options are
+  // in the tab sidebar.
+  useEffect(() => {
+    const targetElement = document.querySelector('#menu-item-mediaArchive');
+
+    if (!targetElement) {
+      return;
+    }
+
+    const toolBars = document.querySelectorAll('.media-frame-toolbar');
+
+    if (!toolBars) {
+      return;
+    }
+
+    if (targetElement.className === 'media-menu-item active') {
+      toolBars.forEach(element => {
+        element.style.display = 'none';
+      });
+    }
+
+    const classChangeCallback = mutationsList => {
+      mutationsList.forEach(mutation => {
+        if (mutation.type === 'attributes' && mutation.attributeName === 'class') {
+          if (targetElement.className === 'media-menu-item active') {
```

Comment:
This all `if` statement could be shortened to
```
toolBars.forEach(element => element.style.display = targetElement.className === 'media-menu-item active' ? 'none' : 'block');
```

Reply:
For some reason if I shorten this, the toolbar no longer appears

Reply:
That's weird 🤔 But it's not a big deal, if it breaks we can leave it as is